packages/frontend/editor-ui/src/features/dataStore/DataStoreView.vue

```diff
@@ -141,6 +133,16 @@ const onCardRename = async (payload: { dataStore: DataStoreResource }) => {
 onMounted(() => {
 	documentTitle.set(i18n.baseText('dataStore.dataStores'));
 });
+
+watch(
```

Comment:
If this is invoked when users land on data stores page, can we just check it in `onMounted` instead of watching?

Reply:
Not only on landing. If you're on the page an click the add datastore button or close it it will still go through this watcher.